search-app/src/cmr/search/routes.clj

```diff
@@ -55,13 +56,21 @@
        [(msg/mixed-arity-parameter-msg mixed-param)]))
     (handler request)))
 
+(defn parse-multipart-body [request]
+  (let [parsed (multipart/multipart-params-request request)]
+    (string/join "&" (for [[k v] (:multipart-params parsed)]
```

Comment:
did ring not have anything for you to do this? None of these would work: https://ring-clojure.github.io/ring/ring.middleware.multipart-params.html#var-parse-multipart-params

Reply:
Hmm ok, the one you suggested did not work, but there is another one that might. Double checking